api/app/serializers/spree/v2/storefront/variant_serializer.rb

```diff
@@ -3,8 +3,22 @@ module V2
     module Storefront
       class VariantSerializer < BaseSerializer
         set_type :variant
-        attributes :name, :sku, :price, :weight, :height, :width, :depth, :is_master,
-          :slug, :description, :track_inventory
+
+        attributes :name, :sku, :price, :currency, :display_price, :weight, :height,
+                   :width, :depth, :is_master, :options_text, :slug, :description,
+                   :track_inventory
+
+        attribute :purchasable do |variant|
+          variant.in_stock? || variant.backorderable?
```

Comment:
I'd advise you to move it to a separate method since we'll need it in products serializer as well therefore it'll be nice to have a possibility of writing something like `variants.any?(&:purchasable?)`

Reply:
@kushniryb nice idea!